blocks/header/header.css

```diff
@@ -1,275 +1,458 @@
-  header nav .nav-hamburger {
-    display: none;
-    visibility: hidden;
-  }
+.hamburger-menu-close .line {
+  width: 100%;
+  height: 0.13rem;
+  background-color: var(--theme-blue);
 }
 
-/* brand */
-header .nav-brand {
-  grid-area: brand;
-  flex-basis: 128px;
-  font-size: var(--heading-font-size-s);
-  font-weight: 700;
-  line-height: 1;
+.hamburger-menu-close .line-1 {
+  transform: rotate(45deg) translateY(50%);
 }
 
-header nav .nav-brand img {
-  width: 128px;
-  height: auto;
+.hamburger-menu-close .line-2 {
+  transform: rotate(-45deg) translateY(-50%);
 }
 
-/* sections */
-header nav .nav-sections {
-  grid-area: sections;
-  flex: 1 1 auto;
+/* Search bar related css */
+.search-bar {
   display: none;
-  visibility: hidden;
-  background-color: var(--overlay-color);
+  flex-direction: row;
+  justify-content: center;
+  font-size: 0.688rem;
+  color: var(--dark-grey);
 }
 
-header nav[aria-expanded="true"] .nav-sections {
-  display: block;
-  visibility: visible;
-  align-self: start;
+.search-bar .category-picker {
+  border: 0.06rem solid var(--normal-grey2);
+  border-radius: 1.875rem 0 0 1.875rem;
+  padding: var(--space-xs) 0 var(--space-xs) 0;
+  min-width: 7.188rem;
+  text-align: center;
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.search-bar .search-box {
+  padding: var(--space-xs) 0 var(--space-xxs) 0;
+  border: 0.06rem solid var(--normal-grey2);
+  border-left: none;
+  border-radius: 0 1.875rem 1.875rem 0;
+  min-width: 17.188rem;
+  text-align: center;
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.search-bar .category-picker .categories {
+  margin-left: 0.938rem;
+}
+
+.search-bar .category-picker .dropdown {
+  font-family: icomoon, sans-serif;
+  margin-right: 0.625rem;
+  font-weight: 800;
+  color: var(--theme-dark-blue);
+  font-size: 0.75rem;
+}
+
+.search-bar .search-box input, input::placeholder {
+  margin-left: 0.938rem;
+  border: none;
+  width: 100%;
+  outline: none;
+  font-size: 0.875rem;
+  color: var(--light-grey);
 }
 
-header nav .nav-sections ul  {
+.search-bar .search-box input:focus{
+  color: var(--theme-dark-blue);
+}
+
+.search-bar .search-box .search-icon {
+  margin-right: 0.938rem;
+}
+
+/* main content related css */
+.main-content {
+  margin-left: 0.938rem;
+  margin-right: 0.938rem;
+}
+
+h1,h2 {
+  text-align: center;
+  color: #053D6E;
+}
+
+p {
+  font: 400 1rem/1.5rem Poppins, sans-serif;
+  color: #4a4a4a;
+}
+
+/* Main content breadcrumbs css */
+.breadcrumbs {
+  font-size: 1rem;
+  margin: 1.25rem 0;
+  color: var(--theme-blue);
+}
+
+.breadcrumbs a {
+  text-decoration: none;
+  color: var(--theme-orange);
+  transition: color 0.3s;
+}
+
+/* Topmost grey navigator related css */
+
+.global-navigator {
+  background-color: var(--lighter-grey);
+}
+
+.global-navigator .container {
+  display: none;
+  overflow-x: auto;
+  font-size: 0.813rem;
```

Comment:
same for fonts-size, weight etc., let's reference a variable here instead of adding color code here.

Reply:
WIP

Reply:
done